tests/codegen/repr/transparent-struct-ptr.rs

```diff
@@ -1,23 +1,36 @@
-//@ revisions: x32 x64 sparc sparc64
+//@ revisions: i686-linux i686-freebsd x64-linux x64-apple wasm32
 //@ compile-flags: -O -C no-prepopulate-passes
-//
-//@[x32] only-x86
-//@[x64] only-x86_64
-//@[sparc] only-sparc
-//@[sparc64] only-sparc64
```

Comment:
`sparc` and `sparc64` both have different ABIs, and neither of them match this file...

(But we never noticed because we don't test sparc.)